CONTRIBUTING.md

```diff
@@ -32,10 +32,10 @@ refcards
 └── etc...
 ```
 
-## Publication des modifications
-La contribution doit passer par une nouvelle branche du repository.
-La branche comportant les modifications sera revue et mergée par un autre collaborateur référent sur la technologie.
+## Changes Publications 
+The contribution must be allong the new branch some repository. 
+The branch make change will be reviewed and merged by another contibutor referent on the technology.
 
-## Revue du contenu
-N'oubliez pas que les RefCards font parti de la vitrine technologie de la société. La revue du contenu permet d'éviter les fautes d'orthographe et de grammaire.
+## Revues
+Don’t forgot that the Refcards they ar part of technology showcase of company. The review it avoided misspelling.
```

Comment:
```suggestion
Don’t forgot that the refcards are parts of company technology showcase. The review allows to avoid some misspellings for example.
```